shanoir-ng-datasets/src/main/java/org/shanoir/ng/dicom/DicomProcessing.java

```diff
@@ -41,4 +47,40 @@ public Attributes getDicomObjectAttributes(DatasetFile image, Boolean isEnhanced
 		}
 	}
 
+    public ExaminationAttributes getDicomExaminationAttributes(Study study, Boolean isEnhanced) throws PacsException {
```

Comment:
Hi @julien-louis, the PacsException in this class are confusing, as all calls get into DicomObjectAttributes, that is read from the disk, if I understood it well. With kind regards, Michael

Reply:
I've changed it to ShanoirException